addons/hide-backpack/button.css

```diff
@@ -0,0 +1,46 @@
+[class*="paint-editor_canvas-controls_"] {
+  /* vertically align zoom buttons with backpack button */
+  align-items: center;
+}
+
+[class*="paint-editor_bitmap-button_"] {
+  align-self: stretch;
+}
+
+[class^="paint-editor_zoom-controls_"] {
+  margin-inline-end: 45px;
+}
+
+.sa-backpack-button {
+  position: absolute;
+  bottom: 13px;
+  width: 36px;
+  height: 36px;
+  display: flex !important;
+  justify-content: center;
+  align-items: center;
+  padding-bottom: 2px; /* vertical position of the icon */
+  background-color: #575e75;
```

Comment:
I think the button and icon should be the same colour as the others. The green can stay though.

Reply:
I chose a different color because the button behaves differently - it's a toggle button.